versions.txt

```diff
@@ -17,3 +17,6 @@ proto-google-iam-v2:1.45.1:1.45.1
 grpc-google-iam-v2:1.45.1:1.45.1
 google-cloud-core:2.49.1:2.49.1
 google-cloud-shared-dependencies:3.41.1:3.41.1
+gapic-showcase:0.0.0:0.0.1-SNAPSHOT
```

Comment:
This is OK as long as release-please does not start bumping versions for showcase. Can you confirm it or maybe @suztomo can confirm?